properties/P000078.md

```diff
@@ -1,10 +1,13 @@
 ---
 uid: P000078
 name: Finite
+aliases:
+  - Cardinality $<\aleph_0$
 refs:
   - mr: MR2048350
     name: General Topology (Willard)
 ---
-A topological space on a set $X$ is said to be finite provided the cardinality of $X$ is finite.
+
+There exists no surjective function from the space onto $\mathbb N$.
```

Comment:
Yeah...  How about just simply "the cardinality of the space is finite"?  People know what that means, no need to make it more involved than that.

Reply:
Addressed in afd393d